pkg/services/ngalert/ngalert.go

```diff
@@ -106,6 +111,12 @@ func (ng *AlertNG) init() error {
 		return err
 	}
 
+	imageService, err := image.NewScreenshotImageServiceFromCfg(ng.Cfg, ng.Metrics.Registerer, store, ng.renderService)
```

Comment:
**TODO**: Should we use `ng.Metrics.Registerer` here?

Reply:
Feels ok to me, given that the service lives in ngalert. How likely to you think it is to make it out into its own non-alerting service?